docs/en/modules/ws2812.md

```diff
@@ -150,6 +150,55 @@ Divide each byte of each led by the given value. Useful for a fading effect
 ```lua
 buffer:fade(2)
 ```
+## ws2812.buffer:fadeIn()
+Multiply each byte of each led by the given value. Useful for a fading effect
+
+#### Syntax
+`buffer:fadeIn(value)`
+
+#### Parameters
+ - `value` value by which to multiply each byte
+
+#### Returns
+`nil`
+
+#### Example
+```lua
+buffer:fadeIn(2)
+```
+## ws2812.buffer:fadeOut()
+Divide each byte of each led by the given value. Useful for a fading effect.
+
+#### Syntax
+`buffer:fadeOut(value)`
+
+#### Parameters
+ - `value` value by which to divide each byte
+
+#### Returns
+`nil`
+
+#### Example
+```lua
+buffer:fadeOut(2)
+```
+## ws2812.buffer:shift()
```

Comment:
Yes, I'd prefer [rotate](https://en.wikipedia.org/wiki/Bitwise_operation#Rotate_no_carry) to distinguish it from [logical shift](https://en.wikipedia.org/wiki/Bitwise_operation#Logical_shift).